stackoverflow.txt

```diff
@@ -0,0 +1,67 @@
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com###sidebar > #hot-network-questions
+! RSS feed links (Recent questions feed (on homepage), Question feed (on question page), User feed (on a user's profile page))
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com###feed-link
+
+! Footer (About, Contant Us, etc.)
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.site-footer
+
+! Profile pictures
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .user-gravatar16
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .user-gravatar32
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .user-gravatar48
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .user-gravatar64
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .gravatar-wrapper-16
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .gravatar-wrapper-32
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .gravatar-wrapper-48
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .gravatar-wrapper-64
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .user-details:style(margin-left:0px!important)
+! Users' points
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .user-details > .-flair
+! Profile pictures on Top Questions page
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.s-post-summary--meta > .s-user-card > .s-user-card--avatar
+! Users' points next to username on Top Questions page
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.s-post-summary--meta > .s-user-card > .s-user-card--info > .s-user-card--awards
+! Hide annoying usernames
+! Put username text to match on (case insensitive) between the //
+!stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.s-user-card--link > .flex--item:has-text(//i)
```

Comment:
What does the `:has-text(//i)` do?

Reply:
@verhovsky

Reply:
From what I understand it is a boilerplate to remove annoying users, that is not currently in use.

Reply:
That's correct, it's just an example of how to do it.